src/registry_schemas/schemas/conversion.json

```diff
@@ -8,46 +8,14 @@
   "title": "Conversion Filing",
   "properties": {
     "conversion": {
-      "required": [
-        "nameRequest",
-        "offices",
-        "parties",
-        "shareStructure",
-        "contactPoint"
-      ],
-      "type": "object",
-      "title": "Conversion Filing",
-      "properties": {
-        "nameRequest": {
-          "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/name_request"
+      "anyOf": [
+        {
+          "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/corps_conversion"
         },
-        "offices": {
-          "registeredOffice": {
-            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
-          },
-          "recordsOffice": {
-            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
-          },
-          "required": [
-            "registeredOffice"
-          ]
-        },
-        "parties": {
-          "type": "array",
-          "items": {
-            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/parties#/definitions/party"
-          }
-        },
-        "shareStructure": {
-          "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/share_structure"
-        },
-        "contactPoint": {
-          "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/contactPoint"
-        },
-        "incorporationAgreement": {
-          "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/agreementType"
+        {
+          "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/firms_conversion"
```

Comment:
What is the path here then? `filing.conversion.firms_conversion` ?

Reply:
I think you are referring to outdated file. Please check the latest